src/Strategies/BootstrapStrategy.php

```diff
@@ -23,6 +23,10 @@ public function __construct(
 
     public function shouldAllow()
     {
+        if (getenv('VAIMO_COMPOSER_PATCHER_ALLOW_GLOBAL_COMMANDS')) {
```

Comment:
Cool approach :) putting it here will allow not only the commands but would also trigger the patches to be applid on composer update and composer install; 

I'd probably advise putting it nearer the [commands provider](https://github.com/vaimo/composer-patches/blob/master/src/Plugin.php#L65) declaration (where it would become alternative to shouldAllow) or rename the variable not to mention "Commands".